tools/transformers/modeling_internlm.py

```diff
@@ -844,6 +844,7 @@ def __init__(self, tokenizer) -> None:
                 self.query = query
                 self.history = history
                 self.response = ""
+                self.chche = []
```

Comment:
cache?

Reply:
cache the tokens generated by the model in each step

Reply:
typo: chche->cache

Reply:
Oh! I corrected this stupid mistake😵